tasks/system/Windows.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Set configset list
+  win_find:
+    paths: '{{ solr_configset_path }}'
+    file_type: directory
+  register: configset_output
+  when: auto_populate_configset_list
+
+- name: Populate configset list
+  set_fact:
+    configset_list: >-
+      {{ configset_output.files | map(attribute='path') | list }}
+  when: auto_populate_configset_list
+
+- name: Get config files list contained replication handler
+  win_command: >
```

Comment:
Why `win_command` instead of `win_find`?

Reply:
win_find can examine file content?